test/EFCore.PG.FunctionalTests/Query/NorthwindAsyncSimpleQueryNpgsqlTest.cs

```diff
@@ -0,0 +1,16 @@
+using Microsoft.EntityFrameworkCore.Query;
+using Microsoft.EntityFrameworkCore.TestUtilities;
+using Xunit.Abstractions;
+
+namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query
+{
+    public class NorthwindAsyncSimpleQueryNpgsqlTest : NorthwindAsyncSimpleQueryTestBase<NorthwindQueryNpgsqlFixture<NoopModelCustomizer>>
+    {
+        public NorthwindAsyncSimpleQueryNpgsqlTest(NorthwindQueryNpgsqlFixture<NoopModelCustomizer> fixture, ITestOutputHelper testOutputHelper)
+            : base(fixture)
+        {
+            Fixture.TestSqlLoggerFactory.Clear();
+            //Fixture.TestSqlLoggerFactory.SetTestOutputHelper(testOutputHelper);
```

Comment:
Yeah, this comes from EF Core. An environment variable wouldn't work because you'd need to restart the IDE to get it. A static field/property would be pretty similar to the existing system... The comment isn't amazing but there's much more important stuff to spend time on...